contracts/modules/capital/Pool.sol

```diff
@@ -361,11 +490,11 @@ contract Pool is IPool, MasterAware, ReentrancyGuard {
     emit NXMBought(msg.sender, ethIn, tokensOut);
   }
 
-  /**
-   * @dev Sell NXM tokens and receive ETH.
-   * @param tokenAmount Amount of tokens to sell.
-   * @param  minEthOut Minimum amount of ETH to be received. Revert if ethOut falls below this number.
-   */
+  /// Sell NXM tokens and receive ETH.
```

Comment:
why change the comment style on top of functions?

the /** */ is pretty standard across multiple languages and looks better/more familiar for most devs.

Reply:
Here's why:

1. You can easily copy or move chunks of the comment without having to worry about turning it back into a comment.
2. It doesn't make any assumptions about smart tooling that handles comments.
3. It's 2 lines smaller.
4. It's also standard across multiple languages.
5. It doesn't affect the code or other devs in any way, even if we're looking in subjective matters such as looks and familiarity. You don't have to stick to it, but it definitely makes my dev experience more pleasent.

Reply:
Understood.

I would want things to be consistent so all comments in 1 way, i prefer the old style. I'll let the others chime in on this one.